frappe/integrations/offsite_backup_utils.py

```diff
@@ -67,7 +67,23 @@ def get_latest_backup_file(with_files=False):
 
 	return database, config
 
+def take_files_backup():
+	"""Only zips and places public and private files in backup folder"""
+	from frappe.utils.backups import BackupGenerator
+
+	odb = BackupGenerator(
+		frappe.conf.db_name,
+		frappe.conf.db_name,
+		frappe.conf.db_password,
+		db_host=frappe.db.host,
+		db_type=frappe.conf.db_type,
+		db_port=frappe.conf.db_port,
+	)
 
+	odb.todays_date = now_datetime().strftime('%Y%m%d_%H%M%S')
```

Comment:
```suggestion
```

https://github.com/frappe/frappe/blob/45ec8ed4857cc2d63e025a72e7d4a6dd568755c1/frappe/utils/backups.py#L97-L121

Let's move this inside `BackupGenerator.set_backup_file_name` instead. It doesn't seem to be used anywhere else either.

Reply:
Done, also moved todays_date value into set_backup_file_name